cluster-install/install.sh

```diff
@@ -172,11 +173,14 @@ get_configs() {
 
     # ------------ IMAGE TAG ------------
     # Docker TAG environment variables
+    # __DEFAULT_CONNECTOR_OPERATOR_IMAGE_TAG is set via Github Actions before adding this script to the Git Release
+    readonly __DEFAULT_CONNECTOR_OPERATOR_IMAGE_TAG=""
```

Comment:
If you call the install script from a branch, the CONNECTOR_OPERATOR_IMAGE_TAG will now be mandatory. This will prevent people from installing an out of date operator.